.github/workflows/build_static.yml

```diff
@@ -12,6 +12,9 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
+  RUSTFLAGS: "--cfg async_executor_impl=\"async-std\" --cfg async_channel_impl=\"async-std\""
```

Comment:
@elliedavidson @rob-maron would it be possible to have HotShot build with a default async runtime?

Reply:
We could probably change some of the conditional compilation attributes to use `async-std` if none are specified. Would that be an alright solution? Other than that, I don't think it's possible to set default `cfg` values without editing something like `.cargo/config.toml`.

Reply:
yeah that sounds good!

Reply:
@DieracDelta Can you do this update when you update the profiles in our justfile? :)